src/config/networks/fuse.ts

```diff
@@ -13,7 +13,7 @@ import { fuseBalancesHandler } from 'src/logic/safe/api/fetchTokenCurrenciesBala
 const baseConfig: EnvironmentSettings = {
   clientGatewayUrl: 'https://safe-service.fuse.io/cgw/v1',
   txServiceUrl: 'https://safe-service.fuse.io/txs/api/v1',
-  gasPrice: 1e9, // 1 Gwei TODO: add gasPriceOracles
+  gasPrice: 11e9, // 1 Gwei TODO: add gasPriceOracles
```

Comment:
it's not working with 10 gwei?

Reply:
It works, but I see that metamask recommends 11 gwei

Reply:
10 gwei should be enough.
I saw some comments from the safe team, are they relevant to the PR?

Reply:
Yes, it's about making the safeTxGas = 0